contracts/templates/FairSaleTemplate.sol

```diff
@@ -56,44 +64,43 @@ contract FairSaleTemplate {
         mesaFactory = IMesaFactory(ISaleLauncher(_saleLauncher).factory());
         saleTemplateId = _saleTemplateId;
 
-        uint256 orderCancelationPeriodDuration = 100;
-        uint256 minimumBiddingAmountPerOrder = 100;
         bool isAtomicClosureAllowed = false;
         tokenSupplier = _tokenSupplier;
         tokenOut = _tokenOut;
-        tokenOutSupply = _tokenOutSupply;
+        tokensForSale = _tokensForSale;
 
         encodedInitData = abi.encode(
             IERC20(_tokenIn),
             IERC20(_tokenOut),
-            orderCancelationPeriodDuration,
+            _orderCancelationPeriodDuration,
             _duration,
-            uint96(_tokenOutSupply),
+            uint96(_tokensForSale),
             _minBuyAmount,
-            minimumBiddingAmountPerOrder,
+            _minimumBiddingAmountPerOrder,
             _minRaise,
             isAtomicClosureAllowed
         );
 
-        initialized = true;
-
         emit TemplateInitialized(
-            _tokenOut,
             _tokenIn,
+            _tokenOut,
             _duration,
-            _tokenOutSupply,
+            _tokensForSale,
             _minPrice,
             _minBuyAmount,
-            _minRaise
+            _minRaise,
```

Comment:
```suggestion
            _minimumRaise,
```

We should follow the same convention among contracts. We use `_minimumRaise` in `FixedPriceSale.sol`, but `minAllocation` and `maxAllocation` in various places.